.github/services/hdfs/hdfs_default_azblob/action.yml

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: hdfs_default_azblob
+description: 'Behavior test for hdfs default over azure blob storage'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup java env
+      uses: actions/setup-java@v4
+      with:
+        distribution: temurin
+        java-version: "11"
+    - name: Load secrets
+      uses: 1password/load-secrets-action@v1
+      with:
+        export-env: true
+      env:
+        # Need a environment variable with this name to securely encrypt them - otherwise there will be warning
+        # See https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/CredentialProviderAPI.html#Keystore_Passwords
+        HADOOP_CREDSTORE_PASSWORD: op://services/hdfs/credstore_password
```

Comment:
Need a environment variable with name `HADOOP_CREDSTORE_PASSWORD` to securely encrypt the credentials. If this variable is not set, we can still encrypt the credentials with default password, but not so secure.

Reference: https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/CredentialProviderAPI.html#Keystore_Passwords

Reply:
Can you explain more why we need to care about this in test CI?

Reply:
The credentials such as `OPENDAL_AZBLOB_ACCOUNT_KEY` are encrypted and saved to `/tmp/hdfs/wasb.jceks` in the ci machine's local file system, so that hdfs can pick up the credentials. If we don't specify `HADOOP_CREDSTORE_PASSWORD`, the encrypting key will be a default value that is publicly known.

If no one has chance to read from `/tmp/hdfs/wasb.jceks` in ci machine, this issue will be fine and we don't need `HADOOP_CREDSTORE_PASSWORD`. But otherwise, they will be able to decrypt the real credentials

Reply:
> If no one has chance to read from `/tmp/hdfs/wasb.jceks` in ci machine, this issue will be fine and we don't need `HADOOP_CREDSTORE_PASSWORD`. But otherwise, they will be able to decrypt the real credentials

Yes, no one can read. We trust our committers to not allow such thing happen.